test/test_torch.py

```diff
@@ -3757,6 +3757,261 @@ def test_tril(self):
         torch.tril(x, out=res2)
         self.assertEqual(res1, res2, 0)
 
+    def test_tril_indices(self):
+        # test 1 X 1 matrix
+        ti = torch.tril_indices(1, 1)
```

Comment:
Just curious, does `device=cuda` work right now? `torch.tril_indices(1, 1, device='cuda')`. I'm wondering if our code generation automatically generates that code as well